builder/src/main/java/cz/xtf/builder/db/AbstractSQLDatabase.java

```diff
@@ -77,5 +78,32 @@ public SQLExecutor getSQLExecutor(String hostname, int port) {
                 getPassword());
     }
 
+    @Override
+    protected void configureContainer(ContainerBuilder containerBuilder) {
```

Comment:
So the main question is. Is this change backward compatible?

This is `protected`, so I assume you have updated all occurences in xtf. But in theory `AbstractSQLDatabase` can be used in any of testsuites. But in that case its overrided configureContainer should be used. So this seems to me fully  backward compatible.

Am I missing something?

Reply:
There are 3 classes inheriting from `AbstractSQLDatabase` and each was overriding `configureContainer` from `AbstractDatabase` in xtf - 
* `PostgreSQL` and `MySQL` are now using this one, both methods did the same thing with different params
* `Derby` now instead of overriding `AbstractDatabase#configureContainer` overrides `AbstractSQLDatabase#configureContainer`, which will not change behaviour
* `Derby` now has inherits `getProbeSettings`, but it will raise an exception if called. `Derby` does not call it and I cant imagine new class inherited from `PostgreSql` which would call `configureContainer` without its own `ProbeSettings`. It has to specify command to check readiness with.

I'm pretty sure this is fully backwards compatible.